ouija.js

```diff
@@ -316,6 +316,8 @@ function updatePostFlair(post, response){
 //awesome workaround from https://mathiasbynens.be/notes/javascript-unicode
 //for getting accurate character count even when handling emojis
 function countSymbols(string) {
+	// Do not count emoji variations and modifiers, http://unicode.org/reports/tr51/index.html
+	string = string.replace(/[\u{20E0}\u{20E3}\u{FE0E}\u{FE0F}\u{1F3FB}-\u{1F3FF}]/ug,'');
```

Comment:
1. It restricts the text after "goodbye" to a period. Users occasionally put commentary there, which leads to more goodbye messages and biased/slower resolution.

2. I tested both changes in my browser but I don't have node.js so you might re-check. I checked out ouijabot to debug why "⚡️" wasn't getting accepted and found that `Array.from("⚡️").length > 1`.